apps/client/resources/public/stylesheets/main.css

```diff
@@ -116,7 +116,9 @@ div#more-info {
   margin-bottom: 20px;
 }
 
-div#more-info a {
+div#more-info a,
+div.missing-keys > p > a,
+#cta > div.missing-keys > p > a {
```

Comment:
Do we need line 121? this styling should already be captured by 120, am unsure if we need to say it again or specifically divs within `#cta` elements.

Reply:
This was left around when I was trying to find a selector that works. I think I'll remove the one on line 120

Reply:
Updated it b701f54